boutdata/data.py

```diff
@@ -110,6 +110,13 @@ def __setitem__(self, key, value):
             return
         self._keys[key.lower()] = value
 
+    def __contains__(self, key):
+        """
+        Check if a key is in the section
+        """
+        key = key.lower()
+        return key in self._sections or key in self._keys
```

Comment:
On a side note:
Is `self._key` redundant of `self._section`? I.e. could they be merged in to one in the future?

Reply:
> Is self._key redundant of self._section? I.e. could they be merged in to one in the future?

I don't think so - self._sections are the nested `BoutOptions` objects (subsections), while self._keys are key-value pairs for the actual options. It is possible to do e.g. `"section" in opt.sections()` to check just the sections, or `"key" in opt.values()` to check just the options.

Looking again, there's something that confuses me now:
```
from boutdata.data import BoutOptions

opt = BoutOptions()
opt["a"] = 1
opt.getSection("mysection")

"A" in opt.values() # True
"A" in list(opt.values()) # False
"A" in opt.keys() # False - this one because keys() uses list(opt.values()) internally
```
I don't understand how `"A" in opt.values()` manages to be `True`...